apps/core-app/src/legos/form.ts

```diff
@@ -0,0 +1,23 @@
+import { CustomFormLego } from './config';
+import { FIELD } from './fields';
+import { TX } from './tx';
+
+export const FORM: Record<string, CustomFormLego> = {
+  SHARE_SWAP: {
+    id: 'SHARE_SWAP',
+    title: 'Swap Tokens for Shares',
+    subtitle: 'Token Proposal',
+    description:
+      'Request membership or increased stake in the DAO. Any tribute must be available in your wallet when proposal is executed.',
+    fields: [FIELD.TITLE, FIELD.DESCRIPTION, FIELD.LINK],
+  },
+  SIGNAL: {
+    id: 'test',
+    title: 'Test Form',
+    subtitle: 'This is a test form',
+    description: 'This is a test form',
+    requiredFields: { title: true, description: true },
+    tx: TX.POST_SIGNAL,
+    fields: [FIELD.TITLE, FIELD.DESCRIPTION, FIELD.LINK],
```

Comment:
does it need an expiration too? or should singal default to no expiration?

Reply:
So for expiration currently, seeing as we're awaiting the input, I have it so that it will search the value automatically from any value '.formData.expiry'. And if there isn't a value, it will default to two weeks from proposal submit. I can also default to 0 as we discussed earlier on Monday.